tests/test_snapshots.py

```
@@ -334,3 +338,161 @@ def test_write_snapshots_handles_errors() -> None:
+    )
+
+
+def test_raw_events_excluded_when_false() -> None:
+    """events.json is not written when include_raw_events is False."""
+    storage = MockStorage()
+    storage.contributions = [
+        ContributionEvent(
+            github_user="alice",
+            event_type="pr_merged",
+            repo="org/repo",
+            created_at=datetime(2024, 1, 15, tzinfo=timezone.utc),
```

Ruff (0.15.7)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)